docstr_coverage/cli.py

```diff
@@ -220,7 +233,12 @@ def execute(paths, **kwargs):
     if kwargs["badge"]:
         badge = Badge(kwargs["badge"], total_results["coverage"])
         badge.save()
-        print("Docstring coverage badge saved to {!r}".format(badge.path))
+
+        if kwargs["verbose"]:
+            print("Docstring coverage badge saved to {!r}".format(badge.path))
+
+    if kwargs["percentage_only"] is True:
+        print(total_results["coverage"])
```

Comment:
@fabiosangregorio, do you think we could run into problems because we `raise SystemExit(1)` below, after printing the coverage, when coverage < `kwargs["fail_under"]`? Normal usage seems fine since raising SystemExit(1) doesn't output anything. Just want to make sure it won't mess with the badge workflow

Reply:
@HunterMcGushion I think everything will be fine, because raising `SystemExit(1)` in the Github CI will fail the CI build, and I am fetching only successful builds from the Github APIs when rendering the badge.